src/interfaces/wasm_bindgen/api.rs

```diff
@@ -123,3 +125,147 @@ pub async fn webassembly_upsert(
         .await
         .map_err(|e| JsValue::from(format!("During Findex upsert: {e}")))
 }
+
+/// See [`FindexSearch::search()`](crate::core::FindexSearch::search).
+///
+/// # Parameters
+///
+/// - `master_key`              : master key
+/// - `label_bytes`             : bytes of the public label used for hashing
+/// - `keywords`                : list of keyword bytes to search
+/// - `max_results_per_keyword` : maximum results returned for a keyword
+/// - `max_depth`               : maximum recursion level allowed
+/// - `progress`                : progress callback
+/// - `fetch_entries`           : callback to fetch from the Entry Table
+/// - `fetch_chains`            : callback to fetch from the Chain Table
+#[wasm_bindgen]
+#[allow(clippy::too_many_arguments)]
+pub async fn webassembly_search_cloud(
+    token: String,
+    label_bytes: Uint8Array,
+    keywords: ArrayOfKeywords,
+    max_results_per_keyword: i32,
+    max_depth: i32,
+    fetch_chains_batch_size: i32,
+    base_url: Option<String>,
+) -> Result<SearchResults, JsValue> {
+    let mut findex_cloud = FindexCloud::new(token, base_url)?;
+    let master_key = KeyingMaterial::<MASTER_KEY_LENGTH>::try_from_bytes(
+        findex_cloud.token.findex_master_key.as_ref(),
+    )
+    .map_err(|e| JsValue::from(format!("While parsing master key for Findex upsert, {e}")))?;
+
+    let label = Label::from(label_bytes.to_vec());
+
+    let keywords: HashSet<_> = Array::from(&JsValue::from(keywords))
```

Comment:
You shouldn't need to explicit the type:
```suggestion
    let keywords = Array::from(&JsValue::from(keywords))
```

Reply:
Good catch, fixed in 0be88d8